arrow-avro/Cargo.toml

```
@@ -40,6 +40,8 @@ default = ["deflate", "snappy", "zstd", "bzip2", "xz"]
 deflate = ["flate2"]
 snappy = ["snap", "crc"]
 canonical_extension_types = ["arrow-schema/canonical_extension_types"]
+md5_alg = ["md5"]
```

@nathaniel-d-ef Nit, but you could also just do this:

```suggestion
md5 = ["dep:md5"] # or even md5 = []
```